### DISCUSSION TITLE
Create Rectangle with a given size (paper Size)

### DISCUSSION THREAD

**Discussion by surffan** [OpenOrienteering/mapper#2062] [Questions & Answers] [answered]
Hi all. I'm a beginner and now working on a first map.
I should become a map of a camping ground with the surrounding area for a scout camp.

First I imported an export of open street map and added an orthophoto and a map as template.
Now I'm working an the map itself.
My target is to print the map and the sign explanation on a A4 Paper.

In the printing preview i can see that it should work.

![Druckvorschau](https://user-images.githubusercontent.com/707767/165901104-0e2e1201-1d06-4233-96fd-0a7dfbd3973b.png)

But I like also to see my target paper size while painting the map.

My Idea is now to paint a rectangle an a separate map part with the size of the paper.
So I created a new symbol and a draw a rectangle.

![MapDrawingWithPaperBoundary](https://user-images.githubusercontent.com/707767/165901110-11e0a651-932f-4ee4-841e-0172e001d8eb.png)

**But what is an easy way to create this rectangle in the size of DIN A4 210 x 297 mm ?**

Or is there an other better way to visualize the target paper size while drawing the map?

Cheers Michael

**Comment by dl3sdo** [comment]
Hi Michael,
unfortunately there is currently no way to manually set point coordinates or manually define the size of objects.
I run into the same situation when I cut the map by defining a rectangle in order to get the right size for having it printed (normally you
would need to make it a bit smaller unless you have a printer that prints 'borderless').
But when you draw the rectangle, try to use the normal 'S' (Pfade zeichnen) tool instead the rectangle tool. Press Ctrl to get 90° angles. The tool will show you the size in meters (and thus implicitly the paper size) while drawing.
Regards, Matthias

**Reply by surffan** [reply]
Hi Matthias

Thanks for the quick answer. The Idea to draw two lines with the path tool with the desired length was a good workaround.
With this helping lines with the right size it is much easier to draw the elements like titles on the map and decide where I stop drawing.

For printing then i hide my paperborder symbol an all is fine.

![MapDrawingWithPaperBoundary_exact](https://user-images.githubusercontent.com/707767/165922376-b675416c-7d5b-43d0-9794-e1a549d93595.png)

So now its time for some first fieldwork to get more details in the forest area and get the right amount of information in the building area.

**Reply by dl3sdo** [reply]
Michael, I can send you some slides (in German) which I titled as "Erste Schritte / Tipps & Tricks" regarding the OOM usage, if you send me an email to oom AT kuehlewein PUNKT de.

**Comment by mlerjen** [comment]
You could define a point symbol representing the paper outline. At the symbol definition you indeed can work with exact values.

**Reply by dl3sdo** [reply]
@mlerjen: you are right of course. I had overseen this possibility if it's just for seeing the paper dimensions.
You cannot use that symbol for cutting the map but you could then use this symbol as a template for drawing a rectangle.

**Comment by arashipl** [comment, marked as answer]
There is another simple option I like - using built-in grid options.
In configure grid I set "Show grid", Positioning to "milimeters on map" and Horizontal and Vertical spacing to paper dimensions.
Then I can use any linear symbol to draw a rectangle using snap to grid (shift+click). 
This results in a precise dimensions with a minimum effort. 
![image](https://user-images.githubusercontent.com/37872952/176163483-cf672c0c-9220-4857-8afd-77b96e3a47f6.png)
